packages/common/src/store/purchase-content/sagas.ts

```diff
@@ -1160,6 +1155,17 @@ function* watchStartPurchaseContentFlow() {
   yield takeLatest(startPurchaseContentFlow, doStartPurchaseContentFlow)
 }
 
+function* watchEnsureUserBankExists() {
+  yield takeEvery(ensureUserBankExists, function* () {
```

Comment:
nit: slight misnomer as it doesn't actually ensure the user bank exists, but tries to create it. it's just an early call to create the user bank, could be just `createUserBankInBackground` or `earlyCreateUserBank` (not great either)

Reply:
Hehe, I had the same thought and I couldn't think of a good name. But I'll update it.